kubeflow/apps/pipelines/pull-upstream.sh

```diff
@@ -13,12 +13,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+#
+# Pulls manifests for Kubeflow Pipelines into the ./upstream folder
+# Arguments:
+#   $1 must be a Git repository. E.g. https://github.com/kubeflow/pipelines.git
+#   $2 must be a TAG or Commit SHA sum. E.g. 2.0.0-alpha.4
+# Output:
+#   Print messages for debugging
 set -ex
 
+if [ $# -eq 0 ]
```

Comment:
NIT: Number of parameters should be 2, so `$# -ne 2 ` seems more accurate.

Reply:
Fixed. Thank you!